packages/boilerplates/ICanManageBoilerplates.ts

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Dolittle. All rights reserved.
+*  Licensed under the MIT License. See LICENSE in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { BaseBoilerplate, Boilerplate } from "./internal";
+
+/**
+ * Responsible for managing boilerplates
+ *
+ * @export
+ * @interface ICanManageBoilerplates
+ */
+export interface ICanManageBoilerplates {
```

Comment:
Are there multiple implementations of these managers? If not, `ICan...` should not be the in the name. We're lacking guidance on this - have a task to add this.